dozer-types/src/arrow_types/from_arrow.rs

```diff
@@ -142,8 +146,41 @@ macro_rules! make_text {
     }};
 }
 
+fn make_json(column: &ArrayRef, row: &usize) -> Result<DozerField, FromArrowError> {
+    let array = column.as_any().downcast_ref::<array::StringArray>();
+
+    return if let Some(r) = array {
+        let s: DozerField = if r.is_null(*row) {
+            DozerField::Null
+        } else {
+            match JsonValue::from_str(r.value(*row)) {
+                Ok(j) => DozerField::Json(j),
+                Err(_) => DozerField::from(r.value(*row)),
```

Comment:
`r.value(*row)` is an element in the `StringArray` so it will be in string value. If that value is not able to be converted to `JsonValue` by function `from_str` we take the element `r.value(*row)` as a string itself and create string `Field` value using that line by doing `DozerField::from(r.value(*row))`

Reply:
Do you think it's reasonable if we return the deserialization error here?

Reply:
hmm, no. Actually, instead i think it would be better if we just do `DozerField::Json(JsonValue::String(r.value(*row)))`. what do you think?

Reply:
let me add `JsonDeserializationError` in `FromArrowError` then